themes/gatsby-theme-blog/README.md

```diff
@@ -0,0 +1,52 @@
+<p align="center">
+  <a href="https://www.gatsbyjs.org">
+    <img alt="Gatsby" src="https://www.gatsbyjs.org/monogram.svg" width="60" />
+  </a>
+</p>
+<h1 align="center">
+  The Gatsby blog theme
+</h1>
+
+A Gatsby theme for creating a blog.
+
+## Installation
+
+### Use the blog theme starter
+
+This will generate a new site that pre-configures use of the blog theme.
+
+```sh
+gatsby new my-themed-blog https://github.com/gatsbyjs/gatsby-starter-blog-theme
+```
+
+### Manually add to your site
+
+```sh
+yarn add gatsby-theme-blog
```

Comment:
Just to confirm: are we recommending `yarn` here for theme authoring purposes?

Reply:
No, we can change this to npm.